### PR TITLE
Add Gradle properties to the `GradleProject`

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we introduce API for adding Gradle properties to a test Gradle build managed by our `GradleProject`.

Gradle properties are passed in the command line, along with other arguments.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

I'm OK with waiting until #644 is merged if it won't take too long.

**Review by alexander-yevsyukov — APPROVED** [review]
LGTM

**Pull request merged**
